docs/user_guide/hopsworks/apiKeys.rst

```diff
@@ -0,0 +1,38 @@
+=================
+API keys
+=================
+
+An API key allow a user or a program to make API calls
+without having to authenticate with username and password.
+To access an end-point using an API key a client should send the access token
+using the ApiKey authentication scheme.
+
+.. code-block:: bash
+
+   GET /resource HTTP/1.1
+   Host: server.hopsworks.ai
+   Authorization: ApiKey <api_key>
+
+Generate an API key
+~~~~~~~~~~~~~~~~~~~~
+
+To generate an API key choose Settings from the top right dropdown
+menu and goto API keys tab. Give the new API key a unique name and
+add a scope to limit the access rights given to the key.
```

Comment:
"add a scope" is not concrete enough. 
add a scope to limit the access rights given to the key. Adding a scope means that you select the APIs that you want to allow this key to access - e.g., 'serving' and 'project' if this key will be used to access a model being served by Hopsworks.